.ci/lanl/gitlab-darwin-ci.yml

```diff
@@ -76,6 +99,35 @@ test:intel:
     name: "$CI_JOB_NAME-$CI_COMMIT_REF_NAME"
     expire_in: 1 week
 
+test:ibm:
+  stage: test
+  tags: [darwin-slurm-shared]
+  variables:
+    SCHEDULER_PARAMETERS: "-ppower9 -t 1:00:00 -N 1 --ntasks-per-node=16"
+  dependencies:
+    - build:ibm
+  needs: ["build:ibm"]
+  script:
+    - pwd
+    - ls
+    - module load ibm
+    - export PATH=$PWD/install_test/bin:$PATH
```

Comment:
Are you sure the `$PWD` for `test:ibm` side starts in same top-dir as the `build:ibm`.  It seems most likely, but that's the only question I'd have to double check.  Otherwise looks good.

Reply:
yes it is retained as an artifact of the ompi install.  I need for you to approve this PR to be able to merge it.